presto-sqlserver/src/main/java/com/facebook/presto/plugin/sqlserver/SqlServerPlugin.java

```diff
@@ -13,13 +13,23 @@
  */
 package com.facebook.presto.plugin.sqlserver;
 
-import com.facebook.presto.plugin.jdbc.JdbcPlugin;
+import com.facebook.presto.spi.Plugin;
+import com.facebook.presto.spi.connector.ConnectorFactory;
+import com.google.common.collect.ImmutableList;
+
+import static com.google.common.base.MoreObjects.firstNonNull;
 
 public class SqlServerPlugin
-        extends JdbcPlugin
+        implements Plugin
```

Comment:
* All changes related to metadata, binders plugin scaffolding in the sql server plugin should go into separate commit.
* also, why no longer `extends JdbcPlugin`?